homeassistant/components/device_tracker/__init__.py

```diff
@@ -5,34 +5,37 @@
 https://home-assistant.io/components/device_tracker/
 """
 import asyncio
-from datetime import timedelta
 import logging
 import os
-from typing import Any, Sequence, Callable
+from datetime import timedelta
+from typing import Any, Callable, Sequence
+
+import async_timeout
 
 import voluptuous as vol
 
+import aiohttp
+
+import homeassistant.helpers.config_validation as cv
```

Comment:
Please don't sort imports or clean up code not related to your PR. It makes this PR very hard to code review.